.github/CODEOWNERS

```diff
@@ -236,6 +236,7 @@ packages/react-components/react-storybook-addon-codesandbox @microsoft/fluentui-
 packages/react-components/babel-preset-storybook-full-source @microsoft/fluentui-react-build
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27972&commitId=1b1ecfd398f06841bb4b6187a3eee2e761ed05d9&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Image Example Rtl.chromium.png|21656| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>